chapters/en/chapter13/2.mdx

```diff
@@ -0,0 +1,233 @@
+
+3. **Fine-tune the LLM with RL:** Now we use the reward model as the environment for our LLM agent. The LLM generates responses (actions), and the reward model scores these responses (provides rewards). In essence, we're training the LLM to produce text that our reward model (which learned from human preferences) thinks is good.
+
+From a general perspective, let's look at the benefits of using RL in LLMs:
+
+| Benefit | Description |
+|---------|-------------|
+| Improved Control | RL allows us to have more control over the kind of text LLMs generate. We can guide them to produce text that is more aligned with specific goals, like being helpful, creative, or concise. |
+| Enhanced Alignment with Human Values | RLHF, in particular, helps us align LLMs with complex and often subjective human preferences. It's hard to write down rules for "what makes a good answer," but humans can easily judge and compare responses. RLHF lets the model learn from these human judgments. |
+| Mitigating Undesirable Behaviors | RL can be used to reduce negative behaviors in LLMs, such as generating toxic language, spreading misinformation, or exhibiting biases. By designing rewards that penalize these behaviors, we can nudge the model to avoid them. |
+
+Reinforcement Learning from Human Feedback has been used to train many of the most popular LLMs today, such as OpenAI's GPT-4, Google's Gemini, and DeepSeek's R1. There are a wide range of techniques for RLHF, with varying degrees of complexity and sophistication. In this chapter, we will focus on Generalized Policy Rectification Optimization (GRPO), which is a technique for RLHF that has been shown to be effective at training LLMs that are helpful, harmless, and aligned with human preferences.
+
+## Why should we care about GRPO (Generalized Policy Rectification Optimization)?
+
+GRPO (Generalized Policy Rectification Optimization) represents a significant advancement in reinforcement learning for language models. It was developed by DeepSeek and played a crucial role in training their DeepSeek-R1 model, which has achieved state-of-the-art performance on various reasoning benchmarks.
+
+This course is focused on GRPO and its implementation in the DeepSeek-R1 model. But before we dive into the technical details of GRPO, let's look at the benefits of using GRPO over other RLHF techniques.
+
+| Aspect | Description | Technical Significance |
+|--------|-------------|------------------------|
+| Direct Preference Learning | GRPO directly learns from preference data without requiring explicit reward modeling | Eliminates the need for a separate reward model, reducing complexity and potential error sources in the training pipeline |
+| Enhanced Stability | Uses a novel group-based optimization approach that considers multiple responses simultaneously | The group-based approach provides more stable gradients and better convergence properties compared to traditional pairwise comparisons |
```

Comment:
I am not sure if makes sense to mention pairwise comparisons here. Maybe rephrase as "The group-based approach provides more stable gradients and better convergence properties compared to the typical pointwise RL methods"